frb_codegen/src/library/codegen/parser/attribute_parser.rs

```diff
@@ -111,6 +114,7 @@ impl FrbAttributes {
 
 mod frb_keyword {
     syn::custom_keyword!(mirror);
+    syn::custom_keyword!(ignore);
```

Comment:
nit: for these changes, it would be great to make them ordered (e.g. suppose you want to put ignore after `sync`, then ensure it is like that in all places